lib/analysis_backend/command_setter.rb

```diff
@@ -77,5 +83,11 @@ def set_dominance
 			"#{@dominance_obj.set_up_remote}"
 		end
 	end
+    
+    def set_bootstrap
+        if @bootstrap_obj != nil
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer `!expression.nil?` over `expression != nil`.